### PR TITLE
Rule 048 postman testing

### PR DESCRIPTION
**Pull request by Ayo-lextego**

### CODE REVIEW

**Review by vorsterk — CHANGES_REQUESTED** [review]
Hey Ayo please can you use variables for the URLS for arango and the endpoints

ie: const arangourl = pm.environment.get('arangourl');
     '${arangourl}/_db/pseudonyms/_api/cursor'
{{baseUrl}}/function/off-rule-048-rel-1-0-0/execute
{{baseUrl}}/function/off-typology-processor-rel-1-0-0                   etc.....

File: Rule-048.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,9027 @@
...
+									"});",
+									"",
+									"pm.test(\"should have the same outgoing and incoming json\", function () {",
+									"    pm.expect(jsonData.data).to.eql(req);",
+									"});",
+									"",
+									"",
+									"pm.test(\"Remove Seed Data from DB\", function () {",
+									"    pm.sendRequest({",
+									"        url: 'https://frmarango.sybrin.com/_db/transactionHistory/_api/cursor',",
```
Comment (johanfol) [inline Rule-048.postman_collection.json:1472]: please use internal DB names throughout

**Review by johanfol — CHANGES_REQUESTED** [review]

File: Rule-048.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,9034 @@
...
+									"                        REMOVE { _key: transaction } IN transactionHistory",
+									"                        OPTIONS { ignoreErrors: true }`",
+									"            })",
+									"        }",
+									"    })",
+									"});",
+									"",
+									"pm.test(\"Remove Seed Data from transactionGraph\", function () {",
+									"    pm.sendRequest({",
+									"        url: 'https://frmarango.sybrin.com/_db/pseudonyms/_api/cursor',",
```
Comment (uche-lextego) [inline Rule-048.postman_collection.json:1488]: @Ayo-lextego , please change to internal url here.

**Review by vorsterk — APPROVED** [review]

File: Rule-048.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,9005 @@
...
+							"mode": "raw",
+							"raw": "{\r\n    \"query\": \"for transaction In transactionHistory FILTER transaction.FIToFICstmrCdt.GrpHdr.MsgId ==  {{EndToEndId}} return transaction \"\r\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "http://arango.development:8529/_db/transactionHistory/_api/cursor",
```
Comment (johanfol) [inline Rule-048.postman_collection.json:130]: More urls instead of variables.

**Review by johanfol — CHANGES_REQUESTED** [review]

**Review by johanfol — APPROVED** [review]

**Pull request merged**